heal/vlmd/extract/csv_dict_conversion.py

```diff
@@ -0,0 +1,228 @@
+from os import PathLike
+from pathlib import Path
+import re
+
+import pandas as pd
+
+from heal.vlmd.config import JSON_SCHEMA
+from heal.vlmd.extract import utils
+from heal.vlmd.validate.utils import read_delim
+
+
+def convert_datadictcsv(
+    csvtemplate: str,
+    data_dictionary_props: dict,
+    renamemap: dict = None,
+    recodemap: dict = None,
+    droplist: dict = None,
+    item_sep: str = "|",
+    keyval_sep: str = "=",
+) -> dict:
```

Comment:
There are many variable / function names in this file that should be in snake cases but are not. We should change them

Reply:
I think I've updated most of these.